crates/codebase_files/src/lib.rs

```diff
@@ -1,41 +1,53 @@
-use std::path::PathBuf;
-use std::process::{Command, Output};
+use ignore::{WalkBuilder, WalkState};
+use std::path::{Path, PathBuf};
+use std::sync::{Arc, Mutex};
 
+#[derive(Debug, PartialEq)]
 pub struct CodebaseFiles {
     pub paths: Vec<PathBuf>,
 }
 
 impl CodebaseFiles {
     pub fn all() -> CodebaseFiles {
-        let output = Command::new("git").arg("ls-files").output();
-        let mut paths = Self::process_ls_files(output);
-        paths.extend(Self::process_ls_files(
-            Command::new("git")
-                .arg("ls-files")
-                .arg("--others")
-                .arg("--exclude-standard")
-                .output(),
-        ));
+        let mut builder = WalkBuilder::new("./");
+        builder.hidden(false);
+        builder.filter_entry(|entry| entry.path() != Path::new("./.git"));
 
+        let results = Arc::new(Mutex::new(vec![]));
+
+        builder.build_parallel().run(|| {
+            Box::new(|result| {
+                if let Ok(entry) = result {
+                    if entry.file_type().map(|ft| ft.is_file()).unwrap_or(false) {
+                        let mut results = results.lock().unwrap();
+                        let path = entry.path().strip_prefix("./").unwrap_or(entry.path());
```

Comment:
The reason we strip this off right now is because Unused will differentiate between where a token is defined and the different places it's used. It captures where it's defined from the tags file, and it uses this list of files to build out a HashMap of filename -> contents to find matches. Because the paths differ slightly, unused tracks these as separate values and shows the defined/found even though it's technically pointing to the same file.